tutorials/apple-mdm-jamf-setup-guide.mdx

```diff
@@ -44,30 +44,28 @@ In this section, we will set up an MDM profile that instructs devices to establi
 
 ### Configure Smallstep for Jamf
 
-1. In the **Devices** tab, add a device collection and choose **Jamf**
-2. Fill in the details related to your Jamf instance.
-3. In the Devices tab, create a collection
-4. Choose Jamf and select your Jamf instance and a Certificate Authority to use
-5. Click **Accounts** → **Add Account** → **Wifi**
+1. In the **Devices** tab, add a device collection and choose **Jamf Computers** for macOS devices, or **Jamf Devices** for other Apple devices
+2. Fill in your Jamf instance URL and choose **Confirm**
+3. Choose **WiFi**, then **Continue**
+4. Enter your public IP and Wi-Fi SSID. Smallstep needs your public IP address in order to identify your network requests.
+5. The other fields are optional. Select **Continue** when done.
+6. Choose the **Configuration Profile** link on the top right of the accounts page to download a `.mobileconfig` template that you'll import into Jamf.
 
-Smallstep will provide the following values, which you’ll need later:
+<Alert severity="info">
+  <div>
+     Resist the temptation to manually install the `.mobileconfig` file for testing; it won’t work.
+  </div>
+</Alert>
 
-- A Jamf webhook URL, username and password to be used when configuring your Jamf webhook
-- Your root CA certificate, for configuring the `Certificate` payload
-- Your SCEP CA URL, for configuring the `SCEP` payload
-- Your intermediate CA fingerprint, for configuring the `SCEP` payload
+On this page, you will need the **Jamf Settings**:
```

Comment:
I feel like this sentence isn't exactly clear and could be worded better. Which page are we talking about? Are the webhook, username, and password provided by Smallstep or is it to be generated by the user?